validation/api/api.lock.json

```diff
@@ -1106,6 +1131,10 @@
       "public type TimerId"
     ]
   },
+  {
+    "name": "Tuples",
+    "exports": []
```

Comment:
@rvanasa inner modules are not added when running `npm run validate` is this a bug or expected?

Reply:
This is expected due to the way `npm run validate` is currently implemented. We can change this now that we are exposing inner modules. I'm waiting to do this because we might want to simply remove this validation / API lockfile now that the new base library is relatively stable.